src/App.vue

```diff
@@ -110,6 +110,14 @@ export default {
       await this.reloadUserData();
     },
   },
+  metaInfo() {
+    return {
+      // default title
```

Comment:
Let's leave it for the documentation of `vue-meta`

Reply:
ok